action.yml

```diff
@@ -18,7 +18,7 @@ runs:
         bootstrap_version='v1.0.0'
         expected_bootstrap_version_digest='e36a05ab402bfee5463ad4752d8dc2941204c7b01a9a9931f921e91d94ba2484'
         curl -L https://storage.googleapis.com/cosign-releases/v1.0.0/cosign-linux-amd64 -o cosign
-        shaBootstrap=$(shasum -a 256 cosign | cut -d' ' -f1);
+        shaBootstrap=$(sha256sum cosign | cut -d' ' -f1);
```

Comment:
Why don't you replace it in https://github.com/sigstore/cosign-installer/blob/main/action.yml#L42 too ?

Reply:
oh ok, thank you, I missed that one.